spec/utility/entity_merger_spec.rb

```diff
@@ -1,13 +1,17 @@
-# rubocop:disable RSpec/NamedSubject
+# rubocop:disable RSpec/MultipleMemoizedHelpers
 
-describe 'Merging Entities', :merging_helper do
-  subject { EntityMerger.new(source: source_org, dest: dest_org) }
+describe EntityMerger, :merging_helper do
+  subject(:merger) { EntityMerger.new(source: source_org, dest: dest_org) }
 
   let(:source_org) { create(:entity_org, :with_org_name) }
   let(:dest_org) { create(:entity_org, :with_org_name) }
   let(:source_person) { create(:entity_person, :with_person_name) }
   let(:dest_person) { create(:entity_person, :with_person_name) }
 
+  before do
```

Comment:
Why is Link.refresh required in some test suites? (I'm not surprised at by thing kind of hack, but curious what the reason is)

Reply:
In general it shouldn't be necessary, since the view is managed automatically via the Relationship model, but I was getting indeterminate failures in some tests where relationships were in some way implicated, presumably due to some quirk of test setup. Scattering these around the test suite is of course not ideal, but I couldn't think of anything better.